terraform/main.tf

```diff
@@ -0,0 +1,252 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-west-1"
+}
+
+data "aws_vpc" "default_vpc" {
+  default = true
```

Comment:
Only worry here is the co-location using the default VPC. Ideally we would have a specific VPC for the Lynker-spatial products and services. Not an immediate need but certainly an eventual one.

Reply:
Sounds good, should be an easy change! 😄 Though I do have a few questions on resource "ownership" with terraform, so maybe I'll reach out to talk to you about that -- specifically, how should resource creation/state be maintained for "site"-wide resources, like a VPC? (i.e. site in this case being lynker-spatial)